tea.py

```diff
@@ -0,0 +1,674 @@
+import pybamm
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+class Tea(pybamm.BaseModel):
+    """"""
+
+    def __init__(self, parameter_values=None):
+        self.parameter_values = parameter_values
+        if self.parameter_values is None:
+            print(
+                "Warning: Missing parameter values, please supply parameter values to Tea()"
+            )
+        super().__init__()
+
+    def plot_ocvs(self, parameter_values=None):
+        # plot ocv curves, average values against gravimetric capacity of cam and
+        # what if two different parameter sets are used? if, try, except?
+        # in case it is called two times after each other, the second time it will not work
+        # the solution is to call it the second time
+        parameter_values = self.parameter_values
+        try:
```

Comment:
An error will already be raised if the key isn't in `parameter_values`, so no need to do this check.